x-pack/test/api_integration/services/usage_api.ts

```diff
@@ -5,49 +5,39 @@
  * 2.0.
  */
 
-import { TelemetryCollectionManagerPlugin } from '@kbn/telemetry-collection-manager-plugin/server/plugin';
+import { UsageStatsPayload } from '@kbn/telemetry-collection-manager-plugin/server';
 import { FtrProviderContext } from '../ftr_provider_context';
 
 export function UsageAPIProvider({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
-  const supertestNoAuth = getService('supertestWithoutAuth');
 
-  return {
-    async getUsageStatsNoAuth(): Promise<undefined> {
-      const { body } = await supertestNoAuth
-        .get('/api/stats?extended=true')
-        .set('kbn-xsrf', 'xxx')
-        .expect(401);
-      return body.usage;
-    },
-
-    /**
-     * Public stats API: it returns the usage in camelCase format
-     */
-    async getUsageStats() {
-      const { body } = await supertest
-        .get('/api/stats?extended=true')
-        .set('kbn-xsrf', 'xxx')
-        .expect(200);
-      return body.usage;
-    },
+  async function getTelemetryStats(payload: {
+    unencrypted: true;
+    refreshCache?: boolean;
+  }): Promise<Array<{ clusterUuid: string; stats: UsageStatsPayload }>>;
+  async function getTelemetryStats(payload: {
+    unencrypted: false;
+    refreshCache?: boolean;
+  }): Promise<Array<{ clusterUuid: string; stats: string }>>;
```

Comment:
adding these overrides to make it easier to consumers to deal with the output data.